contributing.md

```diff
@@ -1,3 +1,33 @@
+Table of contents:
+- [Contributing to Rekognition](#Contributing-to-Rekognition)
+- [Making a PR](#making-a-pr)
+- [Asking for help](#asking-for-help)
+- [Development environment setup](#Development-environment-setup)
+
+As beginners, navigating the codebase and finding your way out of the documentation can become difficult. This page will help you understand everything about contributing to howdoi and the best practices in open source as well. 
+
+## Contributing to Rekognition
```

Comment:
this should start up with setting up dev environment locally
add link to slack in case someone needs help with setup process

Point them to existing APIs and ask them to try these APIs for themselves. If they find any bugs in the process and/or can come up with a feature request ask them to create an issue for the same

Then point to them the already open issues and provide guidance on how to submit a PR